sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -275,19 +371,7 @@ def _fit(self, y, X, fh):
             for param in self.model.parameters():
                 param.requires_grad = True
         elif self.fit_strategy == "peft":
-            if _check_soft_dependencies(
-                "peft",
-                severity="error",
-                msg=(
-                    f"Error in {self.__class__.__name__}: 'peft' module not found. "
-                    "'peft' is a soft dependency and not included "
-                    "in the base sktime installation. "
-                    "To use this functionality, please install 'peft' by running: "
-                    "`pip install peft` or `pip install sktime[dl]`. "
-                    "To install all soft dependencies, "
-                    "run: `pip install sktime[all_extras]`"
-                ),
-            ):
+            if _check_soft_dependencies("peft", severity="error"):
                 from peft import get_peft_model
```

Comment:
I would suggest you keep the detailed error message, look at the review here https://github.com/sktime/sktime/pull/6457#discussion_r1651325041

Reply:
I am getting error `TypeError: _check_soft_dependencies() got an unexpected keyword argument 'msg'`.

I will use `raise ModuleNotFoundError` to show the message instead.